### PR TITLE
Add check list to PR template

### PR DESCRIPTION
**Pull request by Bento007**
The check list will prevent steps from being missed during the RFC process.

### CODE REVIEW

**Comment by maniarathi** [comment]
Pretty good! I think it's missing some steps around summarizing the discussion after the review periods which is a responsibility of the shepherd.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] **Shepard**: Summarize the review discussion for the Approvers in the top-level summary comment in the RFC pull 
+request
+
+### Oversight Review
+- [ ] **April 8**: Oversight Review Last Call <!--Should be at least 1 weeks-->
+- [ ] Replace "rfc-community-review" with "rfc-oversight-review"
+- [ ] Request Review on Slack Channels. Include a link to the PR, and the last call date.
+  - For *software RFCs*: [#tech-architecture ](https://humancellatlas.slack.com/messages/C6SGU0R3J)
+  - For *governance RFCs*: [#dcp-project-mgmt](https://humancellatlas.slack.com/messages/C53S0GBS7)
+- [ ] Add RFC as an agenda item to next meeting:
```
Comment (Bento007) [inline PULL_REQUEST_TEMPLATE.md:28]: @maniarathi, this is where the Shepard summarizes the discussion.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### Community Review
+- [ ] Add Labels
+  - *For software RFCs*: add the "rfc-community-review" and the appropriate software project name (such as "Data 
+  Store") labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label MUST be the 
+  project name. If uncertain about the appropriate project name, then Ask a PM on the 
+  [HCA #dcp-project-mgmt](https://humancellatlas.slack.com/messages/C53S0GBS7) slack channel
+  - *For governance RFCs*: Add the "rfc-community-review" and the "Governance" labels.
+- [ ] **April 1**: Community Review Last Call <!--Should be at least 2 weeks-->
+- [ ] Request review on Slack channel [#dcp](https://humancellatlas.slack.com/messages/C51A8EDQQ).
+- [ ] **Shepard**: Summarize the review discussion for the Approvers in the top-level summary comment in the RFC pull
```
Comment (maniarathi) [inline PULL_REQUEST_TEMPLATE.md]: s/shepard/shepherd

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (maniarathi) [reply PULL_REQUEST_TEMPLATE.md:28]: Ahh got it thanks!

**Review by maniarathi — APPROVED** [review]
Nit but otherwise looks great! Thank you!

**Review by diekhans — COMMENTED** [review]
Great addition.  Thanks!.

**Pull request merged**